### PR TITLE
arrange docs after some cli parameters are deleted

### PR DESCRIPTION
**Pull request by changweige**
Because some CLI parameters are removed, we have to arrange the docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,33 +101,16 @@ io.containerd.snapshotter.v1    nydus                    -              ok
 
 ## Quickly Start Container with Lazy Pulling
 
-### Start Container on Node
-
-Containerd can start container with specified snapshotter, so `nerdctl` or `ctr` needs to specify the Nydus snapshotter when start container.
+### Start Container on single Node
 
-A CLI tool [ctr-remote](https://github.com/dragonflyoss/image-service/tree/master/contrib/ctr-remote) is alongside. Use Nydus `ctr-remote` to pull Nydus image or start container based on nydus image.
+Start container using `nydus-snapshotter` using `nerdctl` which has native nydus support.
```
Comment (sctb512) [inline README.md:106]: Using `nerdctl`, which has native nydus support, to start a container via `nydus-snapshotter`？

Comment (changweige) [reply README.md]: Fixed

**Review by sctb512 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -80,22 +79,16 @@ systemctl restart containerd
 Or you can start nydus-snapshotter manually.
 
 ```bash
-# `nydusd-path` is the path to nydusd binary. If `nydusd` and `nydus-image` are installed, `--nydusd-path` and `--nydusimage-path`can be omitted.
+# `nydusd-path` is the path to nydusd binary. If `nydusd` and `nydus-image` are installed and can be searched from `PATH`, `--nydusd-path` and `--nydusimage-path`can be omitted.
```
Comment (imeoer) [inline README.md:82]: Why still `--nydusd-path`, `--nydusimage-path` here?

File: README.md [hunk 3]
```diff
@@ -108,35 +101,18 @@ io.containerd.snapshotter.v1    nydus                    -              ok
 
 ## Quickly Start Container with Lazy Pulling
 
-### Start Container on Node
-
-Containerd can start container with specified snapshotter, so `nerdctl` or `ctr` needs to specify the Nydus snapshotter when start container.
+### Start Container on single Node
 
-A CLI tool [ctr-remote](https://github.com/dragonflyoss/image-service/tree/master/contrib/ctr-remote) is alongside. Use Nydus `ctr-remote` to pull Nydus image or start container based on nydus image.
+Start container using `nerdctl` which has native nydus support with `nydus-snapshotter`.
```
Comment (imeoer) [inline README.md:106]: Maybe note `>= 0.22`.

**Comment by imeoer** [comment]
Good work, please also help to update other docs (in the image-service repo).

File: README.md (hunk 2 above)
Comment (changweige) [reply README.md:82]: Another PR https://github.com/containerd/nydus-snapshotter/pull/339/files is addressing this, which is clean up cli help message

File: README.md [hunk 4]
```diff
@@ -80,22 +79,16 @@ systemctl restart containerd
...
 # The default nydus-snapshotter work directory is located at `/var/lib/containerd-nydus`
 
-$ ./containerd-nydus-grpc \
-    --config-path /etc/nydus/nydusd-config.json \
-    --address /run/containerd-nydus/containerd-nydus-grpc.sock \
-    --nydusd-path /usr/local/bin/nydusd \
-    --nydusimg-path /usr/local/bin/nydus-image \
-    --log-to-stdout
+$ sudo ./containerd-nydus-grpc --config /etc/nydus/config.toml --nydusd-config /etc/nydus/nydusd-config.json \
+--address /run/containerd-nydus/containerd-nydus-grpc.sock --log-to-stdout \
```
[the remark this answers is not in the document]
Comment (changweige) [reply README.md:89]: Yes. Good idea.  We can simplify the default command line further

File: README.md (hunk 3 above)
Comment (changweige) [reply README.md:106]: fixed

**Comment by changweige** [comment]
> Good work, please also help to update other docs (in the image-service repo).

Sure. Will raise an according PR

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (changweige) [reply README.md]: the flag `--address` is removed

**Comment by imeoer** [comment]
@changweige Once resolve the conflict, we can merge it.

**Comment by changweige** [comment]
> @changweige Once resolve the conflict, we can merge it.

It was resolved

**Pull request merged**